microdata_tools/validation/components/unit_type_variables/KOMMUNE.json

```diff
@@ -10412,17 +10368,6 @@
         "validFrom": "2020-01-01",
         "validUntil": null
       },
-      {
-        "code": "5412",
-        "categoryTitle": [
-          {
-            "languageCode": "no",
-            "value": "Dielddanuorri - Tjeldsund"
-          }
-        ],
-        "validFrom": "2020-01-01",
-        "validUntil": null
-      },
```

Comment:
According to KLASS. This is the correct one: 5412 - Dielddanuorri - Tjeldsund. @DanielElisenberg